db/migrate/20160615145518_increase_max_size_of_allowed_emails.rb

```diff
@@ -0,0 +1,5 @@
+class IncreaseMaxSizeOfAllowedEmails < ActiveRecord::Migration
+  def change
+    change_column :communities, :allowed_emails, :text, limit: 16.megabytes - 1
+  end
```

Comment:
I wonder if just implementing change is enough in this case to support migrations to both directions. How would Activerecord Migrations know what's the previous type of the column based upon this? I believe you need separate up and down migrations here.